cosmwasm/contracts/ntt-global-accountant/tests/query.rs

```diff
@@ -32,56 +30,56 @@ fn create_accounts(wh: &fake::WormholeKeeper, contract: &mut Contract, count: us
     }
 }
 
-fn create_transfers(
-    wh: &fake::WormholeKeeper,
-    contract: &mut Contract,
-    count: usize,
-) -> Vec<Transfer> {
-    let mut out = Vec::with_capacity(count);
-
-    let mut vaas = Vec::with_capacity(count);
-    for i in 0..count {
-        let emitter_chain = i as u16;
-        let emitter_address = [i as u8; 32];
-        let sequence = i as u64;
-        let token_chain = emitter_chain;
-        let token_address = [i as u8; 32];
-        let recipient_chain = emitter_chain + 1;
-        let amount = Uint256::from(i as u128);
-
-        let body: Body<Message> = Body {
-            timestamp: i as u32,
-            nonce: i as u32,
-            emitter_chain: emitter_chain.into(),
-            emitter_address: Address(emitter_address),
-            sequence,
-            consistency_level: 0,
-            payload: Message::Transfer {
-                amount: Amount(amount.to_be_bytes()),
-                token_address: Address(token_address),
-                token_chain: token_chain.into(),
-                recipient: Address([i as u8; 32]),
-                recipient_chain: recipient_chain.into(),
-                fee: Amount(Uint256::zero().to_be_bytes()),
-            },
-        };
-
-        let (_, data) = sign_vaa_body(wh, body);
-        vaas.push(data);
-        out.push(Transfer {
-            key: transfer::Key::new(emitter_chain, emitter_address.into(), sequence),
-            data: transfer::Data {
-                amount,
-                token_chain,
-                token_address: token_address.into(),
-                recipient_chain,
-            },
-        });
-    }
-
-    contract.submit_vaas(vaas).unwrap();
-    out
-}
+// fn create_transfers(
```

Comment:
Any reason you're commenting out this test rather the editing or removing? Same with the test below?

Reply:
removed for now but left TODOs